src/views/Login.vue

```diff
@@ -1,57 +1,101 @@
+      </div>
+      <div class="input-container">
+        <label for="lastName" class="label">Last name</label>
+        <input id="lastName" v-model="lastName" class="input">
+      </div>
+      <div class="input-container">
+        <label for="email" class="label">Email</label>
+        <input
+          id="email"
+          v-model="email"
+          :class="['input', ($v.email.$invalid && submitted) || !$v.email.email ? 'error-input' : '']"
+        >
+        <span class="error-label" v-if="!$v.email.email">El email no tiene un formato válido</span>
+        <span class="error-label" v-if="!$v.email.required && submitted">Campo requerido</span>
+      </div>
+      <div class="input-container">
+        <label for="password" class="label">Password</label>
+        <input
+          id="password"
+          type="password"
+          v-model="$v.password.$model"
+          :class="['input', $v.password.$error && ($v.password.required || submitted ) ? 'error-input': '']"
+        >
+        <span class="error-label" v-if="!$v.password.required && submitted">Campo requerido</span>
+        <span
+          class="error-label"
+          v-if="$v.password.$dirty && $v.password.required && !$v.password.hasNumber"
+        >El password debe contener al menos un número</span>
+        <span
+          class="error-label"
+          v-if="$v.password.$dirty && $v.password.required && !$v.password.hasUppercase"
+        >El password debe contener al menos una mayúscula</span>
+      </div>
+      <button class="primary-button">Sign up</button>
+    </form>
     <div class="horizontal-division"/>
-    <button :class="'secondaryButton'">Login</button>
+    <button class="secondary-button">Login</button>
   </div>
 </template>
 
 <script>
+import { required, email } from 'vuelidate/lib/validators'
+import { hasNumber, hasUppercase } from '@/utils/validations.js'
+
 export default {
-  name: "home",
+  name: 'home',
   components: {},
-  data() {
+  data () {
     return {
-      firstName: null,
-      lastName: null,
-      email: null,
-      password: null
-    };
+      firstName: '',
+      lastName: '',
+      email: '',
+      password: '',
+      submitted: false
+    }
+  },
+  validations: {
+    firstName: {
+      required
+    },
+    lastName: {
+      required
+    },
+    email: {
+      required,
+      email
+    },
+    password: {
+      required,
+      hasNumber,
+      hasUppercase
+    }
   },
   methods: {
-    generateObject() {
+    generateObject () {
       const formObject = {
         user: {
           first_name: this.firstName,
           last_name: this.lastName,
           email: this.email,
           password: this.password
         }
-      };
-      return formObject;
+      }
+      return formObject
     },
-    printForm() {
-      console.log(this.generateObject());
+    onSubmit () {
+      this.submitted = true
+      console.log(this.generateObject())
```

Comment:
remove this console log

Reply:
But it is part of what the card asks:
![screenshot from 2019-01-14 22-19-42](https://user-images.githubusercontent.com/22583914/51151730-994b3480-184a-11e9-8ab2-f030616a3498.png)